CHANGELOG.md

```diff
@@ -11,28 +11,29 @@
 
 - Introduced abstraction for getting migrations to make migration bundling easier #2775
 - Allow timestamp with timezone on mssql databases #2724
-- Feature/2690: Multiple migration directories #2735
+- Allow specifying multiple migration directories #2735
 - Allow cloning query builder with .userParams({}) assigned to it #2802
+- Allow chaining of increment, decrement, and update #2740
 - Kill queries after timeout for PostgreSQL #2636
 - Allow table names with `forUpdate`/`forShare` #2834
-- Added `onVal` and the associated `not` / `and` / `or` methods for using values in `on` clauses within joins.
 - Added `whereColumn` and the associated `not` / `and` / `or` methods for using columns on the right side of a where clause.
+- Added support for named unique, primary and foreign keys to SQLite3 #2840
+- Manage TypeScript types internally #2845
 
 ### Bug fixes:
 
-- #2758: Implement fail-fast logic for dialect resolution #2776
```

Comment:
2758 actually never was implemented, 2776 was an alternative solution based on discussion in 2758